### DISCUSSION TITLE
Customizing `wailsbindings.exe` (and other executable) location

### DISCUSSION THREAD

**Discussion by lorrding** [wailsapp/wails#3272] [Q&A] [answered]
Hi,
I've been running into some issues with wails trying to dev and build the app.

I'm on laptop with group policy that prevent me from running any untrusted .exe file outside of the `My-app` folder in my user directory (in `C:\Users\%USER%\MyApp`).
So far, I was able to specify a custom location for the wails.exe and I'm able to run commands like `wails doctor`, `wails dev` or `wails build`.

But when running `wails dev` or `wails build`, wails will call an executable named `wailsbinding.exe` located in `%APPDATA%/Local/Temp/wailsbinding.exe`. I cannot execute this as it is blocked by group policy.

I tried running my app in dev mode with the `-skipbindings` flag, but then I get this error from WebView2 telling me that "Edge is not able to read and write in its data directory: `C:\Users\%USER%\AppData\Roaming\test.exe\EBWebView`".

Is there any setting or flag I could use to specify a custom location for theses executables or am I screwed and should give up ?

I'm using ~Arch~ Windows10 (22H2) btw.
Thanks !

**Comment by leaanthony** [comment, marked as answer]
Yeah, unfortunately, that's just the way it works right now. If it was written in %TEMP% that might be better. If you're willing to do a PR, we'd help you. I'm thinking write to a temp dir, exec, delete it.

**Reply by lorrding** [reply]
Alright thanks!
Unfortunately I'm no expert in go and don't have much time to decide on what tools to choose for my project.
I'll still keep an eye on this repo if by any chance someone finds a way around my issue in the future :)
